src/java.base/share/classes/java/util/ServiceLoader.java

```diff
@@ -422,10 +422,8 @@ public final class ServiceLoader<S>
     // Incremented when reload is called
     private int reloadCount;
 
-    private static JavaLangAccess LANG_ACCESS;
-    static {
-        LANG_ACCESS = SharedSecrets.getJavaLangAccess();
-    }
+    private static final JavaLangAccess LANG_ACCESS =
+            SharedSecrets.getJavaLangAccess();
```

Comment:
An oversight when SL was re-implemented in JDK 9. It will look better if you avoid the line break.